yellowbrick/features/pcoords.py

```diff
@@ -56,14 +59,20 @@ def parallel_coordinates(X, y=None, ax=None, features=None, classes=None,
     classes : list of strings, default: None
         The names of the classes in the target
 
+    normalize : string or None, default: None
+        specifies which normalization method to use, if any
+
+    sample : float or int, default: 1.0
+        specifies how many examples to display from the data
+
```

Comment:
I was following the existing convention in this module where the docstrings in the quick method are abbreviated.  I'll go ahead and make all of them match.